packages/nft-exchange-transactions/__tests__/functional/transaction-forging/__support__/transaction-factory.ts

```diff
@@ -40,4 +41,28 @@ export class NFTExchangeTransactionFactory extends TransactionFactory {
 
         return this;
     }
+
+    public NFTRegisterCollection(nftCollection: NFTInterfaces.NFTCollectionAsset): NFTExchangeTransactionFactory {
```

Comment:
@amacar wouldn't be better if you pull transaction-factory from nft-base, avoiding code duplication?

Reply:
Yes, but then we need to export it from the tests folder.

Reply:
Okej is not that much code, we can leave it as it is.